audio_sync/cli.py

```diff
@@ -126,6 +135,36 @@ def _PlotResults(
   print(output)
 
 
+def _PlotAsciiGraph(
+    latencies, start_time, dots_per_usec=0.01, latency_threshold_secs=0.001):
+  """Plots all latencies with timestamp in an ASCII timeline."""
+  thresh_in_dots = int(latency_threshold_secs * SECS_IN_MSEC * dots_per_usec)
+  for latency in latencies:
+    if math.isnan(latency[1]):
+      usecs = 9999.0
```

Comment:
You may want to use a named constant instead of `9999.0`.

Reply:
Done.